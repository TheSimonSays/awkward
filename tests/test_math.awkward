import m from math


print(m.tan(2));
print(m.asin(1));
print(m.acos(2));
print(m.atan(2));
print(m.pow(2, 2));
