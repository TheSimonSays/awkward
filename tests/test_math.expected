-2.18504
1.5708
0
1.10715
4
